src/patternfly/components/Nav/nav.scss

```diff
@@ -1000,7 +1012,6 @@
   --#{$nav}--c-divider--PaddingLeft: var(--#{$nav}__subnav--c-divider--PaddingLeft);
 
   max-height: var(--#{$nav}__subnav--MaxHeight);
-  padding-bottom: var(--#{$nav}__subnav--PaddingBottom);
```

Comment:
Looks like you also need to remove this var - https://github.com/patternfly/patternfly/blob/171eb5cf7841f9612dce372cf045a72f41676433/src/patternfly/components/Nav/nav.scss#L181

Reply:
I agree with @mcoker's observations and with these changes we should be good to go.